cypress/integration/form_spec.js

```diff
@@ -0,0 +1,40 @@
+describe ('Form user flow', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('should render the appropriate elements', () => {
+    cy.contains('WellNews')
+      .get('.cy-sentiment-question').contains('how are you feeling today?')
+      .get('.cy-happy-btn').should('be.visible')
+      .get('.cy-neutral-btn').should('be.visible')
+      .get('.cy-sad-btn').should('be.visible')
+      .get('.cy-view-synopsis-button').should('be.visible')
+  })
+
+  it('should take you to the news feed when the happy sentiment button is chosen', () => {
```

Comment:
Do these three tests need an intercept?

Reply:
You're right. I'll add one to help avoid rate limiting.